Makefile

```diff
@@ -138,7 +138,7 @@ run-preflight: preflight
 
 .PHONY: run-troubleshoot
 run-troubleshoot: support-bundle
-	./bin/support-bundle ./examples/support-bundle/sample-supportbundle.yaml
+	./bin/support-bundle ./examples/support-bundle/sample-supportbundle.yaml --debug --interactive=false
```

Comment:
Have we the same options for preflight?
If so, should the changes no be applied in both?

Reply:
But do we not need to ensure that when we change support bundle we also go along with the changes into the preflight and the other way arround?

Reply:
> 

Yes. All the CLIs will all be made consistent